### DISCUSSION TITLE
How to make history search mode don't display when empty?

### DISCUSSION THREAD

**Discussion by vunhatchuong** [marlonrichert/zsh-autocomplete#695] [Q&A]
I'm unable to combine `zstyle ':autocomplete:*' default-context history-incremental-search-backward` and `zstyle ':autocomplete:*' ignored-input ''`.

My goal is to not show the history search list when I haven't typed anything yet.
This is supposed to not display the history.
![image](https://github.com/marlonrichert/zsh-autocomplete/assets/20581504/e0d61af1-40dc-492d-941a-9da2cf4e1998)

Also is there a way to change the pattern matching highlight color during history search?

**Comment by marlonrichert** [comment]
```sh
zstyle ':autocomplete:*' default-context history-incremental-search-backward
zstyle ':autocomplete:history-incremental-search-backward:*' min-input 1
```

> Also is there a way to change the pattern matching highlight color during history search?

Not at present. Feel free to open a feature request for this.
